src/storage.cc

```diff
@@ -69,6 +68,25 @@ void Storage::CloseDB() {
   db_ = nullptr;
 }
 
+void Storage::InitTableOptions(rocksdb::BlockBasedTableOptions *table_options) {
+  table_options->format_version = 5;
+  table_options->index_type = rocksdb::BlockBasedTableOptions::IndexType::kTwoLevelIndexSearch;
```

Comment:
> Although rocksdb will do some optimization if format_version is 5, i still want to set those options explicitly.

As i said, i think we should set `data_block_index_type` to BlockBasedTableOptions::kDataBlockBinaryAndHash and `data_block_hash_table_util_ratio` explicitly. Some users may are interested in optimizations and learn those.

Reply:
Oh Sorry, I forgot about this option!

Reply:
I don't think we need to expose it explicitly because it's a more advanced configuration that users generally don't need to set. 
And this configuration is at the same level as the other newly added configurations.